osgp/platform/osgp-throttling-service/src/main/java/org/opensmartgridplatform/throttling/services/Sleeper.java

```diff
@@ -0,0 +1,11 @@
+package org.opensmartgridplatform.throttling.services;
+
+import org.springframework.stereotype.Component;
+
+@Component
+public class Sleeper {
+
+  public void sleep(final long millis) throws InterruptedException {
+    Thread.sleep(millis);
+  }
+}
```

Comment:
Do you need this class? Why not do the _Thread.sleep()_ call in your class directly. To verify this call in your unit tests you can use _org.mockito.Mockito.mockStatic_

Reply:
org.mockito.exceptions.base.MockitoException: It is not possible to mock static methods of java.lang.Thread to avoid interfering with class loading what leads to infinite loops

Reply:
check